05-Jason-Luo/wk04/2-tue/mta.rb

```diff
@@ -0,0 +1,97 @@
+# plan_trip 'N', 'Times Square', '6', '33rd' 
+# This is only a suggested function name and signature.
+
+# `puts` shows output similar to this:
+# "You must travel through the following stops on the N Line: 34th, 28th, 23rd, Union Square."
+# "Change at Union Square."
+# "Your journey continues through the following stops: 23rd, 28th, 33rd."
+# "7 stops in total."
+
+trainlines = {
+    :N => ["Times Square", "34th", "N-28th", "N-23rd", "Union Square", "N-8th"],
+    :L => ["L-8th", "6th", "Union Square", "3rd", "1st"],
+    :S => ["Grand Central", "33rd", "6-28th", "6-23rd", "Union Square", "Astor Place"]
+}
+
+def stationToUnion(SelectLine, SelectUnion, SelectStation)
+    if SelectStation > SelectUnion 
+        return "#{(SelectLine[SelectUnion..SelectStation]).reverse()}\n\n" # when SelectStation > SelectUnion, slice from SelectUnion, then reverse the array to shuffle back to correct order. 

```

Comment:
```suggestion
        return "#{(SelectLine[SelectUnion..SelectStation]).reverse}\n\n"
```
In Ruby I try to leave off parenthesis for method calls wherever I can.